src/sentry/migrations/0393_set_none_date_uploaded_to_date_added.py

```diff
@@ -0,0 +1,60 @@
+# Generated by Django 2.2.28 on 2023-03-22 07:55
+
+import django.utils.timezone
+from django.db import migrations, models
+
+from sentry.new_migrations.migrations import CheckedMigration
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+
+def copy_date_added_to_date_uploaded(apps, schema_editor):
+    ArtifactBundle = apps.get_model("sentry", "ArtifactBundle")
+
+    for bundle in RangeQuerySetWrapperWithProgressBar(
+        ArtifactBundle.objects.filter(date_uploaded__isnull=True)
+    ):
+        bundle.date_uploaded = bundle.date_added
+        bundle.save(update_fields=["date_uploaded"])
+
+
+class Migration(CheckedMigration):
+    # This flag is used to mark that a migration shouldn't be automatically run in production. For
+    # the most part, this should only be used for operations where it's safe to run the migration
+    # after your code has deployed. So this should not be used for most operations that alter the
+    # schema of a table.
+    # Here are some things that make sense to mark as dangerous:
+    # - Large data migrations. Typically we want these to be run manually by ops so that they can
+    #   be monitored and not block the deploy for a long period of time while they run.
+    # - Adding indexes to large tables. Since this can take a long time, we'd generally prefer to
+    #   have ops run this and not block the deploy. Note that while adding an index is a schema
+    #   change, it's completely safe to run the operation after the code has deployed.
+    is_dangerous = False
+    checked = False
```

Comment:
With @beezz we saw the SQL generated with checked=True and he suggested to put it to False because we were expecting to have the SET NOT NULL in the ALTER. Maybe we were missing something but the SQL generated with checked=True is different.

Reply:
Exactly, that was the result.